mulog-kinesis/src/com/brunobonacci/mulog/publishers/kinesis_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns com.brunobonacci.mulog.publishers.kinesis-test
+  (:require [com.brunobonacci.mulog :as μ]
+            [cheshire.core :as json]
+            [cognitect.aws.client.api :as aws]))
+
+(def KINESIS-LOCAL-STREAM-NAME  "mulog-test-stream")
+(def KINESIS-LOCAL-SETTINGS  {:api  :kinesis
+                              :endpoint-override {:protocol :http :hostname "localhost" :port 4568}})
+
+(def local-kinesis (aws/client KINESIS-LOCAL-SETTINGS))
```

Comment:
this client is created a compile-time.
Maybe it is better to have a small macro like `with-test-client` and/or `with-test-steam`